src/libraries/System.Private.CoreLib/src/System/UIntPtr.cs

```diff
@@ -18,8 +19,9 @@ namespace System
 {
     [Serializable]
```

Comment:
The same feedback should be applied in this file as well.

Reply:
believe ive resolved them all